app/models/video.py

```diff
@@ -0,0 +1,20 @@
+from flask import current_app
+from app import db
+from datetime import datetime 
+
+class Video(db.Model):
+    video_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String(200))
+    release_date = db.Column(db.DateTime) 
+    total_inventory = db.Column(db.Integer) 
+    available_inventory = db.Column(db.Integer)
+    renter = db.relationship('Rental', backref='customer', lazy=True)
+    
```

Comment:
Like with `Customer` you can compute `available_inventory` with a relationship:

```suggestion
    rentals = db.relationship('Rental', back_populates='video', lazy=True)
```